### ISSUE TITLE
Add a extra dynamic graph to the homepage

### ISSUE THREAD

**Issue by niconoe** [inbo/vespa-watch#413]
I think that would be some sort of heatmap, right?

- Remark by @peterdesmet: let's think about the map placement to avoid bloating the user interface
- To be implemented by Jasmijn, with the help of @niconoe

**Comment by peterdesmet** [comment]
Assuming we don't need to be able to link to the map, I think it could be solved by using tabs:

<img width="958" alt="Screenshot 2021-05-07 at 11 10 25" src="https://user-images.githubusercontent.com/600993/117427061-ea542a80-af24-11eb-946c-3dced3186296.png">

**Comment by jrhillae** [comment]
Yes, that is a good idea. 

Probably, we will show observations per municipality (as often done with corona maps), as management is regulated per municipality. Example:

![image](https://user-images.githubusercontent.com/35692395/118663895-59693300-b7f1-11eb-932a-bf6979e0b2e7.png)


Also, in Wallonia observations from previous years are colored grey, maby we can add this feature to the map illustrating the spread? Both Karel and Dominique suggested this. So depending on how much work this is, we could implement that.

**Comment by jrhillae** [comment]
Would it be possible to have three tabs in total? Spread, hotspots and management? To start, they can be linked to a folder where I regularly update the file? Is an html suitable? For example:

-example of hotspots:  Vespa-Watch\Website (oud)\images\data_2020\example_hotspots.html

- example of management: Vespa-Watch\Website (oud)\images\data_2020\example_management.html

**Comment by niconoe** [comment]
@jrhillae : yep, I like the idea of having 3 tabs that show different things.

About the implementation: I'm not convinced HTML is good exchange format here, I suggest one of the following approaches instead:

**1. ideal approach**

The JavaScript-based map is populated via a data format to be agreed upon (probably based on JSON). The JSON data would be returned dynamically from an API endpoint implemented in Django. We may use manually-generated and uploaded JSON files instead similarly to what you suggest, but I am not sure the effort would be drastically different (I suspect the frontend/js part would be the harder, and it's the one that would be done in both cases)

**2. Lightweight, short-term approach**

You generate and upload two static image (JPG, PNG) for hotspots and management, and the frontend just load/display those images when the tab is clicked.

**Comment by jrhillae** [comment]
The first option seems the best.  Currently, the figures are interactive (easy to zoom in and out and when pointing to a municipality the name and number of nests appears).  When applying option 2 this will not longer be the case, but it might be a short-term solution.

**Comment by jrhillae** [comment]
@niconoe: option 2 would be ideal on the short term. It is ok for me to implement option 1 on the long term.

**Comment by jrhillae** [comment]
Temporarily, I have (with the help of Sander) created a link 'distribution in 2021' on the homepage, which displays the extra maps. In the long term, we still plan to incorporate these maps to the homepage itself, as extra tabs.
